routing/edge_estimator.cpp

```diff
@@ -25,8 +25,14 @@ inline double TimeBetweenSec(m2::PointD const & from, m2::PointD const & to, dou
 {
   ASSERT_GREATER(speedMPS, 0.0, ());
 
+  // Current time estimation are too optimistic.
+  // Need more accurate tuning: traffic lights, traffic jams, road models and so on.
+  // Add some penalty to make estimation of a more realistic.
+  // TODO: make accurate tuning, remove penalty.
+  double constexpr kTimePenalty = 1.8;
+
   double const distanceM = MercatorBounds::DistanceOnEarth(from, to);
-  return distanceM / speedMPS;
+  return kTimePenalty * distanceM / speedMPS;
```

Comment:
Not sure that it's a good idea to put this constant here as it doesn't affect calculations of the shortest path.